scripts/install.sh

```diff
@@ -447,7 +447,20 @@ function get_latest_version() {
 
     # get latest version directly from website if there is no versions from api
     if [[ -z "${versions}" ]]; then
-        curl --retry 5 --connect-timeout 5 --max-time 30 --retry-delay 0 --retry-max-time 150 -s https://github.com/SumoLogic/sumologic-otel-collector/releases | grep -oE '/SumoLogic/sumologic-otel-collector/releases/tag/(.*)"' | head -n 1 | sed 's%/SumoLogic/sumologic-otel-collector/releases/tag/v\([^"]*\)".*%\1%g'
+        curl \
+        --connect-timeout 5 \
+        --max-time 30 \
+        --retry 5 \
+        --retry-delay 0 \
```

Comment:
Seems like we should either not specify retry-delay (in which case it uses a default) or it should be set to greater than zero?

Reply:
yeah, a 5 second delay sounds reasonable. The default is exponential backoff we probably don't need that.